server/src/main/scala/za/co/absa/atum/server/api/service/DatabaseService.scala

```diff
@@ -38,4 +37,13 @@ class DatabaseService @Autowired()() {
     } yield checkpoint
   }
 
+  /**
+   * Function to retrieve checkpoint based on the provided fields
+   *
+   * @param filterCriteria JSON object containing the fields for filtering the checkpoint
```

Comment:
what is this filterCriteria about? I saw it elsewhere as well

Reply:
It has being changed to `partitioningInfo`.